ToDo-SwiftUI/ViewModels/DetailPageViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DetailPageViewModel.swift
+//  ToDo-SwiftUI
+//
+//  Created by Kim Insub on 2022/08/04.
+//
+
+import Foundation
+import Combine
+import CoreData
+
+final class DetailPageViewModel: ObservableObject {
+
+    @Published var userInput = ""
+    @Published var todoList: Array<TodoEntity> = []
+    @Published var inProgressTodos: Array<TodoEntity> = []
+    @Published var doneTodos: Array<TodoEntity> = []
+
+    var category = CategoryEntity()
+
+    func getAllTodos(context: NSManagedObjectContext) {
+        
+        let response = CoreDataManager.shared.getAllTodos(of: category, context: context)
+        todoList = response ?? [TodoEntity()]
+
+        inProgressTodos = todoList.filter({ todo in
+            todo.isDone == false
+        })
```

Comment:
```suggestion
        inProgressTodos = todoList.filter { $0.isDone == false } 
```

이렇게 한 줄로 바꿔서 쓸 수 있답니다 .. ! 후후